index.js

```diff
@@ -0,0 +1,57 @@
+const omit = require('lodash/omit');
+const { Container, config, format, transports } = require('winston');
+
+const { combine, colorize, label: addLabel, printf } = format;
+
+const container = new Container();
+const TEST = process.env.NODE_ENV === 'test';
+
+let LEVEL = 'info';
+
+const appendMeta = format(i => {
+  const meta = omit(i, ['level', 'message', 'label']);
+  if (!Object.keys(meta).length) return i;
+  i.message = `${i.message}\t${JSON.stringify(meta, null, 2)}`; // eslint-disable-line
+  return i;
+});
+
+class ConsoleTransport extends transports.Console {
+  constructor(opts) {
+    super(opts);
+    this.silent = opts.silent;
```

Comment:
should `silent` be spread out so it doesn't go to the super ctor?

Reply:
its _supposed_ to be supported here so i'm hoping well just remove this at some point